src/aot/aot.cpp

```diff
@@ -26,9 +27,11 @@
 
 #define AOT_ELF_SECTION ".btaot"
 static constexpr auto AOT_MAGIC = 0xA07;
-static constexpr auto AOT_SHIM_NAME = "bpftrace-aotrt";
 static constexpr auto AOT_SECDATA_TEMPFILE = ".temp_btaot";
 
+#define elf_error(fmt, ...)                                                    \
+  fprintf(stderr, "%s: " fmt ": %s.\n", __func__, ##__VA_ARGS__, elf_errmsg(-1))
```

Comment:
Curious, what is this doing? Why don't we want to use our current logging framework? Also shouldn't macros be all CAPS?

Reply:
Whoops, I copied this over from Daniel's libelf example just to get it to work, but forgot to clean it up. Will switch to LOG(ERROR); thanks for the catch.